### PR TITLE
Added Cowardly to two instances of Lion in html

### PR DESCRIPTION
**Pull request by Chadwell-Clark**

### CODE REVIEW

**Comment by Chadwell-Clark** [comment]
@Chadwell-Clark
Added Cowardly to two instances of Lion in html

**Review by SidneyCrandall — APPROVED** [review]
sounds good.

File: index.html [hunk 1]
```diff
@@ -44,7 +44,7 @@ <h3>Dorothy</h3>
 							</div>
 						</div>
 						<div class="friend">
-							<h3>Lion</h3>
+							<h3>Cowardly Lion</h3>
 							<div>
 								<img src="images/lion.jpg" alt="Lion">
 							</div>
```
Comment (bvinson23) [inline index.html:50]: Would you want to change the alt to "Cowardly Lion" as well?

**Review by bvinson23 — CHANGES_REQUESTED** [review]
Added a comment about the alt on the image.

File: index.html (hunk 1 above)
Comment (Chadwell-Clark) [reply index.html:50]: yes definitely! Thanks

**Review by SidneyCrandall — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Chadwell-Clark) [reply index.html:50]: Changed the alt. Thanks for the headsup!

Comment (Chadwell-Clark) [reply index.html:50]: Changed the alt. Thanks for the headsup!

**Review by bvinson23 — APPROVED** [review]
Awesome 👍

**Pull request merged**